### ISSUE TITLE
Portrait Probleme

### ISSUE THREAD

**Issue by 2ndHendrix** [hs-furtwangen/GIS-SoSe-2020#2]
Mein Portrait wurde angezeigt, nun aber nicht mehr obwohl ich im Code nichts verändert hab.

Woran kann das liegen. Der Link und alles stimmt, denn es wurde mir davor ja auch angezeigt.

**Comment by Plagiatus** [comment]
Wir bei mir noch angezeigt, kein Problem bei mir.
![grafik](https://user-images.githubusercontent.com/7681159/80623195-215a4b00-8a4a-11ea-8eb6-7acd879217a8.png)

**Comment by 2ndHendrix** [comment]
Hallo, ich meinte nicht das Bild des Steckbriefs, sondern der Aufgabe01.html

Dort habe ich das Bild verlinkt und es wurde mir auch angezeigt, jetzt aber
nichtmehr.

Lukas Scheuerle <notifications@github.com> schrieb am Mi., 29. Apr. 2020,
18:49:

**Comment by Plagiatus** [comment]
Dann bitte einmal den Link zur ausführbaren Datei (GitHub Pages Link, github.io) um die es geht, und den repository link (github.com) zur Datei um die es geht.

**Comment by 2ndHendrix** [comment]
https://2ndhendrix.github.io/GIS-SoSe-2020/Aufgabe01/index.html

Am Mi., 29. Apr. 2020 um 19:51 Uhr schrieb Lukas Scheuerle <
notifications@github.com>:

**Comment by 2ndHendrix** [comment]
Hat sich erledigt, danke!!!


Am Do., 30. Apr. 2020 um 16:30 Uhr schrieb Manuel Hubert <
2ndhendrix@gmail.com>:

**Comment by Plagiatus** [comment]
Dann bitte den Issue schließen.

**Comment by 2ndHendrix** [comment]
Hallo Lukas,

ich habe leider erst gerade gesehen, dass meine Ampel auf Gelb gesprungen
ist und das ich in allen Dokumenten 12 html Tags hätte verwenden müssen,
dies war mir aus der Aufgabenstellung, die ich gründlich durchgelesen hab,
nicht ersichtlich. Leider habe ich das Praktikum heute verpasst und konnte
somit meine Ampel nicht verteidigen. Gibt es eine Möglichkeit dies jetzt
noch zu verteidigen wenn ich die Aufgabe heute noch komplettiere?

Mit freundlichen Grüßen

Manuel

Am Fr., 1. Mai 2020 um 20:45 Uhr schrieb Lukas Scheuerle <
notifications@github.com>:

**Comment by Plagiatus** [comment]
Wie du in der Aufgabe direkt über dem Q&A Bereich sehen kannst:

> Aufgrund der unklaren Aufgabenstellung wird der Verteidigungszeitraum dieser Aufgabe bis zum 17. Mai um 18:00 verlängert

Also: Ja, kannst du. Unter normalen Umständen allerdings nicht.  
Man kann über eine Verlängerung der Korrekturfrist genauso wie für die Abgabefrist reden, aber eben wenn man sich VOR dem Stichtermin meldet, nicht danach.

**Comment by 2ndHendrix** [comment]
Hallo Lukas,

danke für deine Antwort.

Ich habe die Aufgabe zwei überarbeitet und auf jeder Seite 12 html tags
hinzugefügt sowie die w3-Errors ausgebügelt.

Wie genau kann ich meine Ampel denn nun verteidigen?
Schaust DU es Dir einfach nochmal an oder muss ich dafür etwas spezielles
tun?

mfg
Manuel

Am Fr., 15. Mai 2020 um 11:58 Uhr schrieb Lukas Scheuerle <
notifications@github.com>:

**Comment by Plagiatus** [comment]
Genau, ich schaue einfach am Sonntag nochmal bei allen drüber die noch eine Ampel offen haben. Du brauchst nichts weiter tun wenn du deine Aufgabe korrekt verbessert hast.

**Comment by 2ndHendrix** [comment]
Okay, danke Dir vielmals!
Wenn die ganze Corona Geschichte vorbei ist, lade ich Dich gern mal auf ein
Bier im Engel ein.


Lukas Scheuerle <notifications@github.com> schrieb am Fr., 15. Mai 2020,
22:28:

**Comment by Plagiatus** [comment]
Wenn sich die Frage geklärt hat, bitte den Issue schließen.
